packaging/buildtools/pack.cmd

```diff
@@ -51,9 +51,18 @@ if [%__DotNetHostBinDir%]==[] (goto Usage)
 :: Initialize the MSBuild Tools
 call "%__ProjectDir%\init-tools.cmd"
 
-:: Restore dependencies mainly to obtain runtime.json
+:: Setup deps
 pushd "%__ProjectDir%\deps"
-"%__ProjectDir%\Tools\dotnetcli\bin\dotnet.exe" restore --source "https://dotnet.myget.org/F/dotnet-core" --packages "%__ProjectDir%\packages"
+set __RuntimeJsonVersion=1.0.1-rc2-23931
```

Comment:
I don't think this change is necessary any longer as @eerhardt already added the other location in dir.props to his update script.